clientv3/lease.go

```diff
@@ -114,6 +114,12 @@ type Lease interface {
 	Leases(ctx context.Context) (*LeaseLeasesResponse, error)
 
 	// KeepAlive keeps the given lease alive forever.
+	// When the context "ctx" is canceled or timed out,
+	// returned "LeaseKeepAliveResponse" channel is closed,
+	// and "LeaseKeepAliveResponse" from this closed channel is nil.
+	// If the context is "context.Background/TODO",
```

Comment:
probably wont say anything about the TODO one. line 116 already explains it. we might want to talk about when an error can be returned though.